server/src/main/java/io/druid/metadata/SQLMetadataConnector.java

```diff
@@ -197,9 +197,10 @@ public void createPendingSegmentsTable(final String tableName)
                 + "  \"end\" VARCHAR(255) NOT NULL,\n"
                 + "  sequence_name VARCHAR(255) NOT NULL,\n"
                 + "  sequence_prev_id VARCHAR(255) NOT NULL,\n"
+                + "  sequence_name_prev_id_sha1 VARCHAR(255) NOT NULL,\n"
                 + "  payload %2$s NOT NULL,\n"
                 + "  PRIMARY KEY (id),\n"
-                + "  UNIQUE (sequence_name, sequence_prev_id)\n"
+                + "  UNIQUE (sequence_name_prev_id_sha1)\n"
```

Comment:
It's also not so bad if this does collide, all it'll do is make a sequence that *should* have worked not actually work. So the task trying to do the data load will fail. It won't lead to incorrect data.

I would also eat my hat if it ever happened, ever, to anyone.